test/caniuse.test.js

```diff
@@ -0,0 +1,13 @@
+var browserslist = require('../')
+
+console.warn = jest.fn()
+
+it('test caniuse version doesn\'t throw warning', () => {
+  expect(browserslist.testCanIUse(1526515200000)).toBeUndefined()
```

Comment:
You do not test how `checkLatestVersion` work. It is better to change date inside Browserslist (you can find this method in `last n versions` tests)